### PR TITLE
feat(CarrierLogo): migrate to Tailwind

### PR DESCRIPTION
**Pull request by mvidalgarcia**
Closes FEPLT-1763

 Storybook: https://orbit-mainframev-tw-carrier-logo.surge.sh

### CODE REVIEW

**Comment by mainframev** [comment]
idk why, but visual tests are different in master comparing to this branch 
Screenshot 2023-10-19 at 18 28 52
Screenshot 2023-10-19 at 18 28 57

**Review by mainframev — CHANGES_REQUESTED** [review]

**Comment by mvidalgarcia** [comment]
> idk why, but visual tests are different in master comparing to this branch

I think master is broken, the `max-width: 100%` in the `img` element on the tailwind preflight is messing up with inline logos.

image

If you set it to `max-width: none`, as it's done in the Tailwind commit, many hidden logos show up.

Do you think we should fix this first on master to get the right screenshots?

File: packages/orbit-components/src/CarrierLogo/index.tsx [hunk 1]
```diff
@@ -130,30 +121,49 @@ const CarrierLogo = ({
...
       inlineStacked={inlineStacked}
-      className="orbit-carrier-logo"
     >
       {carriers.slice(0, 4).map((carrierImage, idx) => (
-        <StyledImage
+        <img
+          className={cx(
+            "max-w-none bg-transparent",
+            rounded ? "rounded-circle" : "rounded-normal",
+            inlineStacked && "[&_+_&]:-ms-xs", // equivalent to &:not(:first-child)
```
[the remark this answers is not in the document]
Comment (mvidalgarcia) [reply packages/orbit-components/src/CarrierLogo/index.tsx:146]: TIL! 
Yes yes, that helps a lot! Thanks!

**Comment by mainframev** [comment]
> > idk why, but visual tests are different in master comparing to this branch
> 
> I think master is broken, the `max-width: 100%` in the `img` element on the tailwind preflight is messing up with inline logos.
> 
> image
> 
> If you set it to `max-width: none`, as it's done in the Tailwind commit, many hidden logos show up.
> 
> Do you think we should fix this first on master to get the right screenshots?

This PR already is fixing that, I think we can just wait for merge of this and that will automatically fix the snapshot after

**Comment by mainframev** [comment]
> idk why, but visual tests are different in master comparing to this branch
> 
> Screenshot 2023-10-19 at 18 28 52 Screenshot 2023-10-19 at 18 28 57

@mvidalgarcia I also think that the visual test is missing 3 or 4 carrier logos with inlinestacked and rounded properties, can it be added? 🙏🏻

**Comment by mvidalgarcia** [comment]
@mainframev it was broken on [prod](https://kiwicom.github.io/orbit/?path=/story/carrierlogo--visual-test). It should be fixed [now](https://orbit-mainframev-tw-carrier-logo.surge.sh/?path=/story/carrierlogo--visual-test).

I've already performed the [visual tests](https://www.chromatic.com/build?appId=6523bafdbc6087c220b3cc48&number=135), comparing master with [master fixed](https://github.com/kiwicom/orbit/commit/5c9d19edcf93d19a8294edd0d582e55294f99cd4).

**Comment by mainframev** [comment]
> @mainframev it was broken on [prod](https://kiwicom.github.io/orbit/?path=/story/carrierlogo--visual-test). It should be fixed [now](https://orbit-mainframev-tw-carrier-logo.surge.sh/?path=/story/carrierlogo--visual-test).
> 
> I've already performed the [visual tests](https://www.chromatic.com/build?appId=6523bafdbc6087c220b3cc48&number=135), comparing master with [master fixed](https://github.com/kiwicom/orbit/commit/5c9d19edcf93d19a8294edd0d582e55294f99cd4).

@mvidalgarcia I know it was broken, I mentioned it above. I meant, that there are 3 and 4, but they aren't rounded, could you also add rounded variations?

**Comment by mvidalgarcia** [comment]
@mainframev Ah sorry, I hadn't understood you. 

I've added `inlineStacked rounded` variations.

**Pull request merged**